docs/react-native.md

```diff
@@ -0,0 +1,68 @@
+
+Helix works great with React Native.
+
+React Native, more so than React Web, is
+[finicky](https://github.com/thheller/shadow-cljs/issues/349) about reloading with
+`.forceUpdate`. Harnessing Helix's native Fast Refresh integration is a big win when
+developing with React Native.
+
+## Example Projects:
+
+* [react-native + shadow-cljs + helix](https://github.com/aiba/helix-react-native)
+* [react-native + shadow-cljs + helix + react-navigation](https://github.com/aiba/helix-react-native-nav)
+
+## Getting Started
+
+In general, the best way to get set up with CLJS + React Native is to create a
+`react-native` directory inside your project and have shadow-cljs compile into
+`react-native/target`. As in [https://github.com/thheller/reagent-react-native](this
+example project).
+
+```bash
+$ npx react-native@0.61.5 init MyApp
+$ mv MyApp react-native
+$ cd react-native && echo 'import "./target/index.js";' > index.js
+
+```
+
+For helix, we just need to add it as a shadow-cljs dependency and also add to
+`shadow-cljs.edn`:
+
+```clojure
+:devtools {:preloads [helix.experimental.refresh]
+           :reload-strategy :full}
+```
+
+It's an open question whether `{:reload-strategy :full}` is strictly necessary to
+emulate the React Fast Refresh behavior, but it doesn't seem to hurt.
+
+Then, we just need to hook into Fast Refresh from the main part of our app, as well
+as add `{:helix/features {:fast-refresh true}}` to every component:
```

Comment:
We should add a link to the ["create a custom macro"](https://github.com/Lokeh/helix/blob/master/docs/pro-tips.md#create-a-custom-macro) section in pro-tips.

Reply:
Good call, link to custom macro pro-tip added.